.circleci/config.yml

```diff
@@ -441,3 +444,35 @@ workflows:
               ignore: /.*/
             tags:
               only: /^[0-9]+\.[0-9]+\.[0-9]+$/
+
+  'Build Earth Images':
+    jobs:
+      - docker/publish:
+          docker-password: DOCKER_HUB_PW
+          docker-username: DOCKER_HUB_USER
```

Comment:
Do these need `$` to be read from the environment?

Reply:
@cicdw no actually it's the name of the variable that we're passing to the orb